src/Relationships.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace Fig\Link;
+
+interface Relationships
+{
+    // Relations defined in https://www.w3.org/TR/html5/links.html#links and applicable on link elements
+    const REL_ALTERNATE = 'alternate';
```

Comment:
These should likely have `public` visibility declared, considering we target PHP 7 for this spec.

Reply:
Technically the current version targets PHP 5; it won't target 7 until the 1.1/2.0 type changes go through.  Visibility here would mean bumping the version requirement for the util package.

Probably good to do that with the next release that also uses the 2.0 version of the PSR.